Src/Support/Google.Apis.Auth/OAuth2/ComputeCredential.cs

```diff
@@ -398,12 +399,32 @@ bool IsLinux()
 #endif
             }
 
-            bool IsWindowsGoogleBios() => throw new NotImplementedException();
+            bool IsWindowsGoogleBios()
+            {
+                Logger.Info("Checking BIOS values on Windows.");
+#if NET45 || NETSTANDARD1_3
+                Logger.Debug("System.Management is not supported in net45 and netstandard1.3");
+                return false;
+#else
+                System.Management.ManagementClass biosClass = new ("Win32_BIOS");
+                using var instances = biosClass.GetInstances();
+
+                foreach(var instance in instances) using (instance)
```

Comment:
If we need to dispose of every instance, should we do that regardless of whether we have an early "out"? Or are they only allocated lazily by the enumerator?
(Either way, I'd prefer to use more braces and indentation instead of this formatting, personally.)

Reply:
Changed braces and indetantion, and also, we are now iterating over the whole collection to dispose of all possible instances (unlikely there's more than one though).